_pages/home/updating-to-version-5/checkout-libraries-release-notes.md

```diff
@@ -223,42 +223,82 @@ Spartacus schematics are designed to assist you in your migration and to help ma
 
 Take a look at the [5.0 TypeScript Changes](https://github.com/SAP/spartacus/blob/84e6b462db1f5dca5c3145c5bc535fc5e2b52fe2/docs/migration/5_0-generated-typescript-changes-doc.md) to get a high-level overview of what has changed.
 
-### Configuring checkout that is already configured to load from the old checkout entrypoint '@spartacus/checkout/root' either eagerly or lazily.
+### Migrating the old checkout configuration
 
-To re-iterate, if you are using Spartacus 4.0, the checkout library only contains a single entrypoint (`@spartacus/checkout`), which contains all checkout flows. On the other hand, in Spartacus 5.0, the library has been further decoupled into different business logic features, with the intention of having a smaller bundle size by splitting it into three different entrypoints instead (`@spartacus/checkout/base/root`, `@spartacus/checkout/b2b/root`, or `@spartacus/checkout/scheduled-replenishment/root`).
+To re-iterate, if you are using Spartacus 4.0, the checkout library only contains a single entrypoint (`@spartacus/checkout`), which contains all checkout flows. On the other hand, in Spartacus 5.0, the library has been further decoupled into different business logic features, with the intention of having a smaller bundle size by splitting it into three different entrypoints instead (`@spartacus/checkout/base`, `@spartacus/checkout/b2b`, or `@spartacus/checkout/scheduled-replenishment`).
 
-If your application is already configured to load from `@spartacus/checkout/root`, you would need to update the reference that imports from it by choosing from either of the three different entrypoints as mentioned earlier.
+If your application is already configured to load from `@spartacus/checkout`, you would need to update the reference that imports from it by changing it to `@spartacus/checkout/base`.
 
-However, if there were added configurations that adds some customized logic to checkout, you might consider creating a wrapper module, which allows additional extensions/functionality to work with your customized checkout module.
+However, if there were added configurations that adds some customized logic to checkout, you might consider creating a wrapper module, which allows additional extensions/functionality to work with your customized checkout module. Creating a wrapper module is the recommended way even if there are no additional customizations.
```

Comment:
I think I wasn't clear [here](https://github.com/SAP/spartacus-docs/pull/1505#discussion_r996851047), sorry for that.
What I meant was: if customers are using B2B and / or Repl, it is recommended to use the wrapper modules (regardless of their customizations). 
If customers want only Base Checkout, by default we don't generate a wrapper module (but as soon as customers want to add some customization, they will have to create it).

That being said, I think we don't need to have too many examples here. Only the "big" example (with `imports: [CheckoutModule, CheckoutB2BModule, CheckoutScheduledReplenishmentModule, CustomizedCheckoutModule]`) should be sufficient IMO.